include/graphene-config.h.meson

```diff
@@ -36,9 +40,9 @@ extern "C" {
 # elif defined(GRAPHENE_HAS_ARM_NEON)
 #  define GRAPHENE_USE_ARM_NEON
 #  define GRAPHENE_SIMD_S "neon"
-# elif defined(GRAPHENE_HAS_GCC)
-#  define GRAPHENE_USE_GCC
```

Comment:
`GRAPHENE_USE_GCC` is an exported public symbol, so it cannot be removed.

You can define it alongside `GRAPHENE_USE_INTRINSICS`.